### DISCUSSION TITLE
Advanced Gui Creator

### DISCUSSION THREAD

**Discussion by Qg9** [BG-Software-LLC/SuperiorSkyblock2#1118] [Pending Feature Requests]
Yo, 
I find it a pity that superior skyblock's gui system is so developed but does not allow you to make a little more "custom" gui, you can't make a gui that can be opened with other commands than /is... , you can't execute a command when the player clicks or you can't make a condition to display an item...
We could say that it is not useful for superior skyblock but since the goal is to make a core and that no server opens without a deluxemenu type plugin I think it is doable
Another solution, allow to open the categories of missions directly with a command (that's what blocks me)

I can dev if needed

**Comment by OmerBenGera** [comment]
- you can make menus to opened using another command by making that command an alias of the original command. If you want it to be a command that is not /is, then youll have to use another plugin that makes it so once the custom command is executed, /is admin openmenu is executed instead.
- you can make commands to be exexuted when clicking an item, check out the wiki for more information
- you cannot make conditional items and there isnt a really useful feature. All comditional items that need to be conditionalled are already handled by the plugin.

You shouldnt use SSB's menus for menus that are not related to ssb, as the plugin isnt really suitable for such thing.

**Comment by Lemvs** [comment]
@OmerBenGera you can close it.
